### PR TITLE
OAuth: add docs for disableAutoLogin param

### LINKED ISSUE #38723: Docs: missing oauth bypass info from PR #28398 is not in current docs.
PR 28398 added documentation to docs/sources/auth/overview.md detailing how to bypass OAuth.  It's not in the docs anymore and would be great to have there since I just had a hell of a time  finding out how to do it.

Thanks

### PR DESCRIPTION
**Pull request by mckn**
This PR contains docs for #28357 - `disableAutoLogin` param. It is a new PR since the #28398 never got merged.

Fixes #38723

### CODE REVIEW

File: docs/sources/auth/overview.md [hunk 1]
```diff
@@ -125,6 +125,12 @@ Defaults to `false`.
 oauth_auto_login = true
 ```
 
+If you want to skip automatic OAuth login and sign in with a username and password, then add the `disableAutoLogin` parameter to the login URL. This disables signing in with OAuth.
+
+Example: `grafana.example.com/login?disableAutoLogin` or `grafana.example.com/login?disableAutoLogin=true`
+
+> **Note:** `disableAutoLogin` parameter is available in Grafana 7.4+.
```
Comment (achatterjee-grafana) [inline docs/sources/auth/overview.md:132]:
```suggestion
### Avoid OAuth login

To sign in with a username and password and avoid automatic OAuth login, add the `disableAutoLogin` parameter to your login URL. 

For example:  `grafana.example.com/login?disableAutoLogin` or `grafana.example.com/login?disableAutoLogin=true`
```

Comment (achatterjee-grafana) [reply docs/sources/auth/overview.md:132]: We do not need the note, as it is for an older version.

**Review by achatterjee-grafana — CHANGES_REQUESTED** [review]
Thank you @mckn for adding the doc content. Added review.

**Review by alexanderzobnin — APPROVED** [review]

File: docs/sources/auth/overview.md [hunk 2]
```diff
@@ -121,6 +121,11 @@ Defaults to `false`.
 oauth_auto_login = true
 ```
 
+### Avoid OAuth login
```
Comment (achatterjee-grafana) [inline docs/sources/auth/overview.md:124]:
```suggestion
### Avoid automatic OAuth login
```

**Review by achatterjee-grafana — APPROVED** [review]
Added one suggestion, otherwise LGTM

**Pull request merged**